app/request_handler.py

```diff
@@ -49,7 +49,7 @@ def writeYelpRecords(yelpVenues):
 def writeVenueProviderRecord(yelpID, details):
     try:
         venue = representation.updateRecord(yelpID, **details)
-        for provider, data in venue["providers"].items():
+        for provider, data in list(venue["providers"].items()):
```

Comment:
Is this really necessary to turn this into a list? e.g., what does items() return, if not a list type? I tried this in the python REPL and didn't seem to have any problems.